assisted-installer/bili/README.md

```diff
@@ -0,0 +1,83 @@
+# **Deploying an OpenShift Cluster using BILI workflow**
+
+This document provides a first view at deploying a cluster using bili workflow, which is a subcommand for openshift-install that will generate a custom iso for installing openshift from a set of ZTP-like manifests and in a unattended fashion.
+
+## **Worklow**
+
+### **Requisites**
+
+#### **Packages**
+
+The openshift-install build with agent support needs to be used. In the future, this will be part of the regular install
+
+Additionally, nmstate needs to be installed and Network Manager should have a minimum version of 1.30. This will also be included directly in the agent subcommand
```

Comment:
```suggestion
Additionally, NMState needs to be installed and NetworkManager should have a minimum version of 1.30. In the future, this will also be included directly in the agent subcommand.
```